### PR TITLE
Introducing the brand new travel policy

### PR DESCRIPTION
**Pull request by matteogalliani**
This is the first version of the travel policy.

We need both English and content review. 

It's still WIP:
- [x] I need to make this section coherent with the others, like [Conferences](https://github.com/nebulab/playbook/blob/master/personal-growth/conferences.md). 
- [x] Also, I'm not sure that its position is in the "Processes."
- [x] I need to fix all the links.

#### Todo

- [x] This change needs a public announcement, _I'll update [the agenda of our monthly meeting](https://tadum.app/rhythms/3JAFLu5H6JKDvLrfzXFZL4nf) once merged_.

### CODE REVIEW

File: processes/travel-policy.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+### Team sponsorship
+
+If you're in the conference marketing team, we won't deduct the cost and the days for that
+conference from your yearly budget.
+
+### Conference speaker
+
+Being invited as a speaker is an honor as well as, pairing with our values, a great way to spread
+your knowledge. At the same time, it will give us much visibility. Therefore, we encourage to submit
+speech proposal to sponsored, listed or even non-listed conferences. If you're invited as a speaker,
```
Comment (kennyadsl) [inline processes/travel-policy.md]: I think `talks proposal` is better.

Comment (kennyadsl) [reply processes/travel-policy.md]: It's not clear if the cost of the Conference is on Nebulab or not in this case.

File: processes/travel-policy.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+we won't deduct the cost and the days for that conference from your yearly budget.
+
+## **Team Retreat**
+
+We will organize one Team Retreat per year, lasting one week. We'll know each other in a better way,
+building stronger relationships and mutual trust and improving communication and cooperation.
+Of course, we will have much fun.
+We'll ask you for confirmation with a three-month notice.
```
Comment (kennyadsl) [inline processes/travel-policy.md]: Just a TODO note. Here we can say much more, maybe in a dedicated page for team retreats. I think we can do this later though.

File: processes/travel-policy.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+injuries that require medical treatment.
+
+#### Useful mobile Apps
+
+- **Wi-Fi Finder,** detects free Wi-fi networks.
+- **MetrO, Kinevia** is a free mobile guide to public transport systems worldwide. It provides the
+fastest route to your point of interest considering the day of the week and the time of the day.
+- **Airline companies official App,** allows through your confirmation or ticket number to download
+an e-ticket for Wallet or Pass2U Wallet. Also, you'll have handy information about your flight.
+- **Mytaxi:** is a Taxi app available in 9 countries within the European Union, Italy included.
```
[the remark this answers is not in the document]
Comment (matteogalliani) [reply processes/travel-policy.md]: I'm moving Uber into a separate section under "Useful Information." Also, I'm adding two links to the new Uber section in reimbursement and apps.

**Comment by aldesantis** [comment]
@matteogalliani this should go in Processes IMO, so we should be good on that front.

File: processes/travel-policy.md (hunk 1 above)
Comment (matteogalliani) [reply processes/travel-policy.md]: la riga sotto è
>If you're invited as a speaker,
we won't deduct the cost and the days for that conference from your yearly budget.

File: processes/travel-policy.md (hunk 2 above)
Comment (matteogalliani) [reply processes/travel-policy.md]: Yes, I agree

**Pull request merged**